portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,54 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <header>
+        <div class="homeheader">
+            <h1>Bria Fassler's Portfolio</h1>
+        </div>
```

Comment:
nit: Keep indentation consistent with the rest of the file (two spaces).

*nit is short for nitpicking, which just means this is a super minor fix